api/src/main/java/com/nicknathanjustin/streamercontracts/contracts/ExpiredContractsSqsHandler.java

```diff
@@ -0,0 +1,24 @@
+package com.nicknathanjustin.streamercontracts.contracts;
+
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.cloud.aws.messaging.listener.annotation.SqsListener;
+
+import java.util.Set;
+
+@RequiredArgsConstructor
+@Slf4j
+public class ExpiredContractsSqsHandler {
+
+    @NonNull private final ContractService contractService;
+
+    @SqsListener("ExpiredDonationsSQS-${application.environment}")
+    @SuppressWarnings("unused") //Method is invoked when pulling AWS SQS. Method is not directly called within our application.
```

Comment:
Does something actually enqueue events on to this queue? Or does this function just wake up every 30 minutes and execute some logic? I haven't read much about this QSQ service but can it just be used as an easier way for configuring a cron job?